woocommerce/woocommerce-gutenberg-products-block
# Add side_effects suffix to the Product Gallery Large Image Next Previous Block

## What

I realized that this E2E test does side effects, so adding the `side_effects` suffix in the filename is necessary.

## Why



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure that E2E PW Jobs are green.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
